client/src/App.js

```diff
@@ -112,23 +139,30 @@ export default function App() {
           <QueueDisplay items={queue} leaveQueueFunction={displayExitModal} />
         </div>
       )}
-      <div className='queueForm'>
-        <input
-          type='text'
-          placeholder='Dine initialer'
-          className='textinput'
-          ref={inputRef}
-          onKeyUp={handleKeyPress}
-        ></input>
-        <br></br>
-        <button className='button' onClick={enterQueue}>
+      <form className='queueForm' onSubmit={handleSubmit}>
+        <div>
+          <input
+            type='text'
+            placeholder='Dine initialer'
+            className='textinput'
+            ref={initialsInputRef}
+          />
```

Comment:
I think it looks better with some space between the two input fields.
adding `margin-left: 0.5em;
  margin-right: 0.5em;` to `.textinput` in `App.css` looks like this?
  
![image](https://github.com/user-attachments/assets/e82bcdb3-4e9b-48e4-92fb-c174d641eede)